packages/launchpad/src/checklist-item/index.tsx

```diff
@@ -13,11 +13,6 @@ const ChecklistItem = ( { task, isPrimaryAction }: { task: Task; isPrimaryAction
 	// Display chevron if task is incomplete. Don't display chevron and badge at the same time.
 	const shouldDisplayChevron = ! completed && ! disabled && ! task.badge_text;
 
-	const handlePrimaryAction = () => {
-		localStorage.removeItem( 'launchpad_siteSlug' );
-		actionDispatch && actionDispatch();
```

Comment:
I wonder why the boolean `actionDispatch` check was necessary 🤔

Reply:
Because `actionDispatch` can be `undefined`, at least according to the definition of the `Task` type of which it's a field. You can pass `undefined` as a `<Button onClick>` prop, but if you're wrapping it in your own callback, you must check.

By the way, the `Task` type is defined three times at various places in the Launchpad code. Good follow-up opportunity to clean that up.